.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  extends: [
+    // add more generic rulesets here, such as:
+    // 'eslint:recommended',
+    'plugin:vue/vue3-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
+  ],
+  rules: {
+    // override/add rules settings here, such as:
+    // 'vue/no-unused-vars': 'error'
+  },
+  parser: "vue-eslint-parser",
+  parserOptions: {
+    parser: "@typescript-eslint/parser",
+    ecmaVersion: "latest",
+    sourceType: "module",
+  }
+}
```

Comment:
I use `standard` and `ts-standard` for this sort of thing, though I see that there's special rules for vue in here. `standard` can format as well as lint in the same step. I like reducing the number of different tools I need to write javascript. Just some food for thought. I'm realizing now that this is probably set up for you by the `vue-cli` tool

Reply:
I looked into standard for vue and didn't find a reasonable solution, so went with eslint since there are nice plugins already.